backend/app/api/api_v1/routers/tests/test_search.py

```diff
@@ -140,11 +137,126 @@ def test_multiple_filters(
     } in query_body["query"]["bool"]["filter"]
 
 
-def test_result_order(test_opensearch, monkeypatch, client, user_token_headers):
+@pytest.mark.parametrize("order", [SortOrder.DESCENDING, SortOrder.DESCENDING])
+def test_result_order_score(
+    test_opensearch, monkeypatch, client, user_token_headers, mocker, order
+):
+    monkeypatch.setattr(search, "_OPENSEARCH_CONNECTION", test_opensearch)
+
+    query_spy = mocker.spy(search._OPENSEARCH_CONNECTION, "query")
+    response = client.post(
+        "/api/v1/searches",
+        json={
+            "query_string": "disaster",
+            "exact_match": False,
+            "sort_order": order.value,
+        },
+        headers=user_token_headers,
+    )
+    assert response.status_code == 200
+    query_response = query_spy.spy_return.raw_response
+    result_docs = query_response["aggregations"]["sample"]["top_docs"]["buckets"]
+
+    s = None
+    for d in result_docs:
+        new_s = d["top_hit"]["value"]
+        if s is not None:
+            if order == SortOrder.DESCENDING:
+                assert new_s <= s
+            if order == SortOrder.ASCENDING:
+                assert new_s >= s
+        s = new_s
+
+
+@pytest.mark.parametrize("order", [SortOrder.DESCENDING, SortOrder.DESCENDING])
```

Comment:
Think this should be ASCENDING, DESCENDING. Same with `test_result_order_title` below (but not the score one).